OpenRA.Mods.Common/Traits/Player/EnemyWatcher.cs

```diff
@@ -100,9 +103,11 @@ public void Tick(Actor self)
 
 		void Announce(Actor self, TraitPair<AnnounceOnSeen> announce)
```

Comment:
I really don't want to add this to this PR, since it has nothing to do with the triggers.
I already have a follow-up PR that does that (+ adds the notifications to RA).